packages/manager/modules/pci/src/projects/project/serving/namespace/models/update/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_serving_namespace_models_update_success": "Votre modèle a été actualisé avec succès",
```

Comment:
Is this text validated by content ? In PCI we usually write `a été mis à jour` instead of `a été actualisé`.